### ISSUE TITLE
Discrepancy in loss weights (weight_rgb, weight_dsm) between code and paper

### ISSUE THREAD

**Issue by qikalamore** [EvilGhostY/MMRSSeg#2]
**Description**

Hi, thank you for the great work and for open-sourcing the code!

I have a question regarding the configuration of the loss function weights. I noticed a discrepancy between the implementation in the code and the values mentioned in the paper.

- **In the paper 3.3. Loss function**, it states that the weights for the aux_OI loss and the aux_DSM loss are set to `1`.
- **However, in the code** the parameters `weight_rgb` and `weight_dsm` are set to `0`.

**Code Reference:**

```
class CSFAFormerLoss(nn.Module):
    def __init__(self, ignore_index=255, edge_factor=10.0, n_classes=6, radius=1, alpha=0.01):
        super(CSFAFormerLoss, self).__init__()
        self.main_loss = JointLoss(SoftCrossEntropyLoss(smooth_factor=0.05, ignore_index=ignore_index),
                                   DiceLoss(smooth=0.05, ignore_index=ignore_index), 1.0, 1.0)
        self.aux_loss = SoftCrossEntropyLoss(smooth_factor=0.05, ignore_index=ignore_index)

    def forward(self, logits, labels, dsm):
        if self.training and len(logits) == 7:
            weight1 = 1.0   #0.6
            weight2 = 1.0   #0.3
            weight3 = 1.0   #0.2
            weight_rgb = 0.0
            weight_dsm = 0.0
            out, RGB_Pre1, DSM_Pre1, RGB_Pre2, DSM_Pre2, RGB_Pre3, DSM_Pre3= logits
            loss_main = self.main_loss(out, labels)
            loss_rgb1 =( self.main_loss(RGB_Pre1, labels))* weight1
            loss_rgb2 =( self.main_loss(RGB_Pre2, labels))* weight2
            loss_rgb3 = (self.main_loss(RGB_Pre3, labels))* weight3
            loss_rgb = (loss_rgb1+loss_rgb2+loss_rgb3)*weight_rgb

            loss_dsm1 = (self.main_loss(DSM_Pre1, labels)) * weight1
            loss_dsm2 = (self.main_loss(DSM_Pre2, labels)) * weight2
            loss_dsm3 = (self.main_loss(DSM_Pre3, labels)) * weight3
            loss_dsm = (loss_dsm1 + loss_dsm2 + loss_dsm3) * weight_dsm


            loss = loss_main  + loss_rgb  + loss_dsm

        else:
            loss = self.main_loss(logits, labels)

        return loss

```
